media-libs/libcaca/libcaca-0.99_beta19-r9.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+RUBY_OPTIONAL=yes
+#USE_RUBY=ruby20
+
+inherit autotools ruby-ng flag-o-matic java-pkg-opt-2 mono-env toolchain-funcs multilib-minimal
+
+MY_P=${P/_/.}
+DESCRIPTION="A library that creates colored ASCII-art graphics"
+HOMEPAGE="http://libcaca.zoy.org/"
+SRC_URI="http://libcaca.zoy.org/files/${PN}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2 ISC LGPL-2.1 WTFPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86"
+IUSE="doc imlib java mono ncurses opengl ruby slang static-libs test truetype X"
+RESTRICT="!test? ( test )"
+
+# ruby? ( ruby_targets_${USE_RUBY} )
+REQUIRED_USE=""
+
+# ruby? (  $(ruby_implementations_depend) )
+DEPEND="
+	imlib? ( >=media-libs/imlib2-1.4.6-r2[${MULTILIB_USEDEP}] )
+	mono? ( dev-lang/mono )
+	ncurses? ( >=sys-libs/ncurses-5.9-r3:0=[${MULTILIB_USEDEP}] )
+	opengl? (
+		>=virtual/glu-9.0-r1[${MULTILIB_USEDEP}]
+		>=virtual/opengl-7.0-r1[${MULTILIB_USEDEP}]
+		>=media-libs/freeglut-2.8.1[${MULTILIB_USEDEP}]
+		truetype? ( >=media-libs/ftgl-2.1.3_rc5 )
+	)
+	slang? ( >=sys-libs/slang-2.2.4-r1 )
+	X? (
+		>=x11-libs/libX11-1.6.2
+		>=x11-libs/libXt-1.1.4
+	)
+"
+RDEPEND="java? ( >=virtual/jre-1.8:* )"
+BDEPEND="
+	virtual/pkgconfig
+	doc? (
+		app-doc/doxygen
+		virtual/latex-base
+		>=dev-texlive/texlive-fontsrecommended-2012
+		>=dev-texlive/texlive-latexextra-2012
+		dev-texlive/texlive-latexrecommended
+	)
+	java? ( >=virtual/jdk-1.8:* )
+	test? ( dev-util/cppunit )
+"
+
+DOCS=( AUTHORS ChangeLog NEWS NOTES README THANKS )
+
+PATCHES=(
+	# Fix out of source tests
+	"${FILESDIR}"/${PN}-0.99_beta18-fix-tests.patch
+	# Debian patches
+	"${FILESDIR}/CVE-2018-20544.patch"
+	"${FILESDIR}/CVE-2018-20545+20547+20549.patch"
+	"${FILESDIR}/CVE-2018-20546+20547.patch"
+	"${FILESDIR}/canvas-fix-an-integer-overflow-in-caca_resize.patch"
+	"${FILESDIR}/Fix-a-problem-in-the-caca_resize-overflow-detection-.patch"
+	"${FILESDIR}/100_doxygen.diff"
+	# Fix doxygen docs install, bug 543870
+	"${FILESDIR}/fix-css-path.patch"
+)
+
+pkg_setup() {
+	java-pkg-opt-2_pkg_setup
+	use mono && mono-env_pkg_setup
+}
```

Comment:
both of these necessary? I'd remove java and mono otherwise

Reply:
Nope, they're both masked too:
```
profiles/arch/base/package.use.mask:77:media-libs/libcaca java mono
```

Removing.